insights/client/__init__.py

```diff
@@ -547,6 +547,24 @@ def show_results(self):
             else:
                 raise e
 
+    def show_inventory_deep_link(self):
+        """
+        Show a deep link to this host inventory record
+        """
+        system = self.connection._fetch_system_by_machine_id()
+        if system is not None and system:
+            if len(system) == 1:
+                try:
+                    id = system[0]["id"]
```

Comment:
It looks like fetch_system_by_machine_id returns a dict, not a list.  Could probably skip the length check above too since a dict length is how many keys are in it. I guess if you get a a system maybe a `if system.get("id")` would be a good check.

Reply:
Oh woops. I thought it was a list.

Reply:
By docstring, it returns a `dict`, but in reality, it returns a `list`, so I'm going to leave this as is.